Sources/FuzzILTool/main.swift

```diff
@@ -16,235 +16,153 @@ import Foundation
 import Fuzzilli
 
 let jsFileExtension = ".js"
-let protoBufFileExtension = ".il.protobuf"
-
-let corpus = Corpus(minSize: 1000, maxSize: 1000000, minMutationsPerSample: 600)
-let jsPrefix = """
-               """
-let jsSuffix = """
-               """
-let lifter = JavaScriptLifter(prefix: jsPrefix,
+let protoBufFileExtension = ".fuzzil.protobuf"
+
+let jsPrefix = ""
+let jsSuffix = ""
+
+let jsLifter = JavaScriptLifter(prefix: jsPrefix,
                 suffix: jsSuffix,
                 inliningPolicy: NeverInline(),
```

Comment:
Should this be "InlineOnlyLiterals()" to match the default policy used by the fuzzer?

Reply:
I specifically chose NeverInline() so that the produced JS matches the provided IL input as closely as possible, to aid sanity-checking the compiler's output. My automated tests are only JS -> IL, so there aren't any issues with breaking what I have currently.

Reply:
Makes sense. There is now also --liftToFuzzIL, which gives you a textual representation of the FuzzIL code. Does that solve that problem as well? In that case I'd change this to `InlineOnlyLiterals()`, if not, I'll leave it `NeverInline()`

Reply:
That works!